specification/newrelic/NewRelic.Management/examples/Monitors_GetMetricStatus_MinimumSet_Gen.json

```diff
@@ -7,12 +7,17 @@
     "resourceGroupName": "rgNewRelic",
     "monitorName": "fhcjxnxumkdlgpwanewtkdnyuz",
     "request": {
+      "azureResourceIds": [
+        "/subscriptions/nqmcgifgaqlf/resourceGroups/rgNewRelic/providers/NewRelic.Observability/monitors/fhcjxnxumkdlgpwanewtkdnyuz"
```

Comment:
>nqmcgifgaqlf [](http://example.com/codeflow?start=24&length=12)

"00000000-0000-0000-0000-000000000000" can be used here."

Reply:
Done